src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,94 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private Node<K, V>[] hashArray;
```

Comment:
```suggestion
    private Node<K, V>[] nodeArray;
```
it's not array of hashes

Reply:
Yep, really. Fixed.